content/docs/getting-started.md

```diff
@@ -21,18 +21,24 @@ If you need to install Go or update your currently installed one, please refer t
 
 You can download the latest source build for containerd from the [Downloads](/downloads) page and then use your favorite process supervisor to get the daemon started.
 
-To install version {{< latest >}}, run the following commands to download and unzip the source code on the target machine.
+### Systemd-based distributions
+To install version {{< latest >}} on a distribution with Systemd, run the following commands:
 
 ```shell
-wget https://github.com/containerd/containerd/archive/v{{< latest >}}.zip
-unzip v{{< latest >}}.zip
+wget https://github.com/containerd/containerd/releases/download/v{{< latest >}}/cri-containerd-cni-{{< latest >}}-linux-amd64.tar.gz.sha256sum
```

Comment:
If this is from the main containerd repo, maybe the same issue is there? This is downloading the hash of the tarball (usually used to verify that the tar.gz you retrieve is exactly what was uploaded), so the next command would fail as the actual tarball itself is not downloaded.

Again, if this came like this from our getting started in the main repo it should be fixed there as well. This step is also repeated below in the "Non-systemd distributions" section.

Reply:
Sorry, i copied the wrong link. :expressionless: Should be fixed now.

Reply:
I don't think you pushed the branch yet? GitHub doesn't show any changes yet since this review.